### PR TITLE
Add units useful to hydrology.

### PR DESCRIPTION
**Pull request by ksheffer-opti**
Includes breaking change: fix spelling of GramPerCubicCentimetre

Sources for conversion factors:
https://en.wikipedia.org/wiki/Cubic_foot#Cubic_foot_per_second
https://en.wikipedia.org/wiki/Acre-foot
https://en.wikipedia.org/wiki/Gallon
https://en.wikipedia.org/wiki/Square_mile
https://en.wikipedia.org/wiki/Siemens_%28unit%29
https://en.wikipedia.org/wiki/Electrical_resistivity_and_conductivity
https://en.wikipedia.org/wiki/Inch_of_mercury
http://www.fws.gov/mountain-prairie/wtr/hydro_def.htm

### CODE REVIEW

File: Source/QuantityTypes/Quantities/Units.csv [hunk 1]
```diff
@@ -107,6 +125,9 @@ Pressure,MillimetreOfMercury,mmHg,133.3224
 Pressure,Bar,bar,1e5
 Pressure,Megapascal,Megapascal,1000000
 Pressure,Atmosphere,atm,101325
+Pressure,InchOfMercury,inHg,3.386389e3
+Pressure,FootOfWater,ftH20,0.433527501928
```
Comment (objorke) [inline Source/QuantityTypes/Quantities/Units.csv]: shouldn't this be ftH2O?

File: Source/QuantityTypes/Quantities/Units.csv [hunk 2]
```diff
@@ -107,6 +125,9 @@ Pressure,MillimetreOfMercury,mmHg,133.3224
 Pressure,Bar,bar,1e5
 Pressure,Megapascal,Megapascal,1000000
 Pressure,Atmosphere,atm,101325
+Pressure,InchOfMercury,inHg,3.386389e3
+Pressure,FootOfWater,ftH20,0.433527501928
+Pressure,InchOfWater,inH20,0.0361272918274
```
Comment (objorke) [inline Source/QuantityTypes/Quantities/Units.csv]: and inH2O?

**Comment by objorke** [comment]
see the 2 comments, besides from that, this looks good!

**Comment by ksheffer-opti** [comment]
Thanks, what a silly mistake! Fixed.

**Comment by objorke** [comment]
Thanks for contributing to the project!

**Pull request merged**